### ISSUE TITLE
Fine control over clipboard function

### ISSUE THREAD

**Issue by helloworld1** [helloworld1/AnyMemo#236]
Original [issue 229](https://code.google.com/p/anymemo/issues/detail?id=229) created by helloworld1 on 2013-06-08T03:33:19.000Z:

<b>What is the feature you want?</b>

<b>How important is it to you?</b>

<b>Do you think other people also want this feature and why?</b>

<b>What is the feature you want?</b>
&quot;&quot;&quot;
Recently, the way copy to clipboard works has changed. Before, questions were copied to the clipboard only when actually testing under Recent. Now, questions are also copied to the clipboard under Open/Preview/edit. This is not helpful. If I come across a new item that I want to add to the database, I should be able to copy it and then go to Anymemo/Open/Preview/edit and create a new record, but the new item is replaced in the clipboard by Anymemo! This forces me to go enter the Preview/edit mode in Anymemo first, and then go back to copy the new item. Please change this back to the way it used to be.
&quot;&quot;&quot;

Basically the customer want to be able to fine control over in [which mode] the question will copy to the clipboard or now.

<b>How important is it to you?</b>
Important to gibosi in http://anymemo.org/forum/viewtopic.php?f=3&amp;t=451

<b>Do you think other people also want this feature and why?</b>
TBD
